libtenzir/builtins/contexts/geoip.cpp

```diff
@@ -475,6 +490,11 @@ class ctx final : public virtual context {
   }
 
   auto reset() -> caf::expected<void> override {
+    if (!mmdb_) {
+      return caf::make_error(ec::lookup_error,
+                             fmt::format("no GeoIP data currently exists for "
+                                         "this context"));
```

Comment:
This function should just be a no-op and always return `{}` in all cases, as the resetting has no effect.